src/components/SerpResultsSortButton.vue

```diff
@@ -17,24 +17,24 @@
           @click="clickOption(option.key)"
           :disabled="menuOptions.length === 1"
       >
-        <v-list-item-icon>
+        <span>
           <v-icon :disabled="menuOptions.length === 1">{{ option.icon }}</v-icon>
```

Comment:
I Converted these deprecated Veutify VListItemIcon components to spans incase we need a wrapper element. So far, I see no need for them and think they could be removed.